src/textual/widgets/_option_list.py

```diff
@@ -477,13 +493,18 @@ def _refresh_content_tracking(self, force: bool = False) -> None:
 
         # Set up for doing less property access work inside the loop.
         lines_from = self.app.console.render_lines
-        options = self.app.console.options.update_width(
-            self.scrollable_content_region.width
-        )
         add_span = self._spans.append
         option_ids = self._option_ids
         add_lines = self._lines.extend
 
+        # Adjust the options for our purposes.
+        options = self.app.console.options.copy().update_width(
```

Comment:
`update_width` does a `copy()` internally, so you won't need the explicit copy.

Reply:
Ahh hah! That explains why it wasn't there before!